octodns_route53/__init__.py

```diff
@@ -28,6 +29,17 @@ def _octal_replace(s):
     return octal_re.sub(lambda m: chr(int(m.group(1), 8)), s)
 
 
+def _healthcheck_ref_prefix(version, record_type, record_fqdn):
+    ref = f'{version}:{record_type}:{record_fqdn}:'
+    # the + 12 is to allow space for the uuid
+    # it is allowing multiple healthchecks per record to exist
+    if len(ref) + 12 > 64:
```

Comment:
Doh, totally neglected to include the `:`

```suggestion
    ref = f'{version}:{record_type}:{record_fqdn}:'
    # the + 13 is to allow space for the uuid and `:`
    # it is allowing multiple healthchecks per record to exist
    if len(ref) + 13 > 64:
```